src/scroll-timeline-css.js

```diff
@@ -72,16 +73,22 @@ function isDescendant(child, parent) {
 }
 
 function createScrollTimeline(name) {
-  const options = parser.scrollTimelineOptions.get(name);
+  let options = parser.scrollTimelineOptions.get(name) ||
+    parser.getViewTimelineOptions(name);
   if (!options) return null;
 
-  const sourceElement = getSourceElement(options.source);
-
-  const scrollTimeline = new ScrollTimeline({
-    ...(sourceElement ? { source: getSourceElement(options.source) } : {}),
-    ...(options.orientation != "auto" ? { orientation: options.orientation } : {}),
-  });
-  return scrollTimeline;
+  if(options.source) {
```

Comment:
We can probably do more upstream processing.  Then this block looks like:

return options.source ? new ScrollTimeline(options) : new ViewTimeline(options);